DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Export/JobProfile/Dynamic modules/Job Profile/Structure/Job Profile.sf

```diff
@@ -6973,11 +6912,42 @@
           "Id": "e839a46c-db69-49ad-abea-698e8ba258be",
           "LastModified": "2020-10-29T15:48:42.667Z"
         },
+        {
+          "ParentTypeId": "63177677-3adb-4cb7-8bd2-136e4c34201f",
+          "FieldNamespace": "Telerik.Sitefinity.DynamicTypes.Model.JobProfile.Skill",
+          "Name": "PSFHidden",
+          "Title": "    PSFHidden",
```

Comment:
We have whitespace here again...

Reply:
yep just to add a gap between title and the checkbox. Doesn't effect the code